src/mmap.rs

```diff
@@ -60,6 +65,8 @@ pub enum MmapError {
 pub struct GuestRegionMmap {
     mapping: MmapRegion,
     guest_base: GuestAddress,
+    file: Option<File>,
+    offset: Option<usize>,
```

Comment:
I think this should be `MemoryRegionAddress`

Reply:
I feel GuestUsize might be more appropriate. I'll change this.

Reply:
@jiangliu @bonzini do you have any preference here?

Reply:
Maybe a (file, off_t) pair is more staightful:)